skll/experiments.py

```diff
@@ -799,6 +805,20 @@ def _classify_featureset(args):
         with open(results_json_path, file_mode) as json_file:
             json.dump(res, json_file, cls=NumpyTypeEncoder)
     else:
+        if results_path:
+            results_json_path = join(results_path,
+                                     '{}.grid_search_cv_results.json'.format(job_name))
```

Comment:
I'd argue that the results file should be named like all the other results files - using the job name rather than this fixed filename. Is there a reason for not doing that?